DirectX11Proj/IScene.cpp

```diff
@@ -3,9 +3,9 @@
 #include "IObject.h"
 
 // Create camera on creation of scene
-IScene::IScene()
+IScene::IScene() : mInitialized(false)
```

Comment:
if you do decide to go with the fast declaration on a new line, change this one as well